CHANGELOG.md

```diff
@@ -31,6 +31,30 @@ This preview version adds support for .NET Core 3.0 WPF and WinForms application
 
 * **[Feature]** Add support for .NET Core 3.0 applications.
 * **[Fix]** Wait to finish processing files before deleting files when calling `SetEnabledAsync(false)` during processing.
+
+### App Center Distribute
+
+#### iOS
+
+* **[Fix]** Obfuscate app secret value that appears as URI part in verbose logs for in-app updates.
+
+### App Center Push
+
+#### Android
+
+* **[Fix]** Fix confusing information log about the availability of the Firebase SDK.
+* **[Fix]** Fix sending the push installation log after delayed start.
+
+### App Center Auth
```

Comment:
Nitpick: we usually sort the services

Reply:
Updated. @MatkovIvan

Reply:
I meant place Auth before Distribute and Push.
Anyway, not big deal